src/instrument/plans/bdp_demo.py

```diff
@@ -316,17 +310,18 @@ def xpcs_bdp_demo_plan(
 
     dm_workflow = DM_WorkflowConnector(name="dm_workflow")
     yield from bps.mv(
```

Comment:
Harder to recognize what is done with the standard style applied.  Better to wrap this with `# fmt: off` & `# fmt: on` to preserve the meaning.